src/modules/vtol_att_control/vtol_att_control_params.c

```diff
@@ -315,3 +315,44 @@ PARAM_DEFINE_INT32(VT_FW_DIFTHR_EN, 0);
  * @group VTOL Attitude Control
  */
 PARAM_DEFINE_FLOAT(VT_FW_DIFTHR_SC, 0.1f);
+
+/**
+ * Target deceleration during backtransition.
+ *
+ * The vehicle will use its pitch angle to try and maintain a target deceleration during the backtransition.
+ * This parameter only applies for standard vtol and tiltrotors.
+ *
+ * @unit m/s2
+ * @min 0.5
+ * @max 5
+ * @decimal 1
+ * @increment 0.5
+ * @group VTOL Attitude Control
+ */
+PARAM_DEFINE_FLOAT(VT_B_DEC_SP, 2.0f);
```

Comment:
In which scenarios does this parameter differ from VT_B_DEC_MSS?

Reply:
@dayjaby I have already prepared a commit which merged those two parameters, will push asap.

Reply:
@dayjaby This is not fixed by the way.